app/src/main/assets/map_theme/jawg/global.yaml

```diff
@@ -1,7 +1,6 @@
 global:
     api_key: false # intended to be overwritten by StreetComplete
     language: false # intended to be overwritten by StreetComplete
-    language_script: Latn # intended to be overwritten by StreetComplete
```

Comment:
Is this change intended?

`global.language_script` is still referenced in this file in line 80 and in

https://github.com/streetcomplete/StreetComplete/blob/18189f123416d9a0b332e50afab58b857452b416/app/src/main/java/de/westnordost/streetcomplete/screens/main/map/components/SceneMapComponent.kt#L69

Reply:
> Is this change intended?

hm, no

Reply:
So this line is always manually added in the StreetComplete repo after copying the map style from the mapstyle repo?

Reply:
I think so

Reply:
With https://github.com/streetcomplete/streetcomplete-mapstyle/pull/129, this is now automated, too (commit [`0fdeda5`](https://github.com/streetcomplete/StreetComplete/pull/4100/commits/0fdeda5fa3aa3a2ae19a510e91ec15ff29591a67) was created by the task).